config/_default/menus/menus.en.yaml

```diff
@@ -2620,6 +2620,11 @@ main:
     parent: static_analysis
     identifier: static_analysis_github_actions
     weight: 103
+  - name: GitHub Source Code Management
```

Comment:
Can we change this to GitHub Pull Requests since this integration is specifically for PRs